build-tools/packages/build-cli/test/commands/generate/buildVersion.test.ts

```diff
@@ -3,7 +3,9 @@
  * Licensed under the MIT License.
  */
 
-import { expect, test } from "@oclif/test";
+// @oclif/test cannot find the path to the project, so as a workaround we configure it explicitly
+import { expect, test as oclifTest } from "@oclif/test";
+const test = oclifTest.loadConfig({ root: import.meta.url });
```

Comment:
Would be nice if this wasn't repeated so much.

Reply:
Not sure how to avoid it. I can move the init code to somewhere shared, but each test file will still have to import it.

Reply:
I refactored this into a shared init function that the test files import and call. I don't think it reduces code, but it does centralize the "workaround" of configuring the oclif test env.

Reply:
Right. I should have also said, "but I don't know how to fix it cleanly." You used to be able to get caller information but that is deprecated as a security concern. Depending on what is needed I thought maybe a file per directory could solve the issue if it was just the or some directory that was needed and not specific file but didn't look into details.